x-pack/plugins/monitoring/server/lib/alerts/fetch_index_shard_size.ts

```diff
@@ -123,7 +117,7 @@ export async function fetchIndexShardSize(
   if (!clusterBuckets.length) {
     return stats;
   }
-
+  const thresholdGB = threshold * gbMultiplier;
```

Comment:
What is the purpose of the `gbMultiplier`? It looks like we multiply the threshold by it here, e.g. 55 * 1000000000 or 55 billion. Then on line 155, we take the average shard size, which has been calculated in bytes (`primaryShardSizeBytes / totalPrimaryShards`) and divide it by this `gbMultiplier`, e.g. 60 billion bytes / 1000000000 would = 60. 

So line 156 would be comparing `if (60 < 55 billion)`, am I reading this right? Do we have a test for this that could verify the assumption?

Reply:
Ah, good catch! Looks like I still had some leftover from doing average on replicas